ci-operator/jobs/openshift/prometheus-operator/openshift-prometheus-operator-master-presubmits.yaml

```diff
@@ -292,6 +292,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build03
+    context: ci/prow/security
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-prometheus-operator-master-security
+    optional: true
+    path_alias: github.com/coreos/prometheus-operator
```

Comment:
I just saw this in the generated job, should we maybe change this in the job config? Iiuc the canonical url is `github.com/prometheus-operator/prometheus-operator` these days?
nothing that needs to block this here though.

Reply:
I'll try to remove it altogether because I'm not sure that it's needed.